pangolin/4.3.1-pdata-1.31/Dockerfile

```diff
@@ -0,0 +1,131 @@
+FROM mambaorg/micromamba:2.0.3-ubuntu22.04 AS app
+
+# build and run as root users since micromamba image has 'mambauser' set as the $USER
+USER root
+# set workdir to default for building; set to /data at the end
+WORKDIR /
+
+# ARG variables only persist during build time
+# had to include the v for some of these due to GitHub tags.
+# using pangolin-data github tag, NOT what is in the GH release title "v1.2.133"
+ARG PANGOLIN_VER="v4.3.1"
+ARG PANGOLIN_DATA_VER="v1.31"
+ARG SCORPIO_VER="v0.3.19"
+ARG CONSTELLATIONS_VER="v0.1.12"
+ARG USHER_VER="0.6.3"
```

Comment:
USHER came out with an update today, but I think we're okay to keep with 0.6.3 because the update shouldn't change lineage calls

Reply:
Angie prepared the global usher tree and everything with usher 0.6.3 so I'm going to stick with it for now. We can check in with her to see if she would recommend upgrading